kumascript/macros/CSSInfo.ejs

```diff
@@ -283,7 +283,7 @@ if (name === "preview-wiki-content") {
     if (!atRule) {
         properties = properties.concat({
             name: "animationType",
-            label: localize(localStrings, "animationType")
+            label: `<a href="/${env.locale}/docs/Web/CSS/CSS_animated_properties">${localize(localStrings, "animationType")}</a>`
```

Comment:
Thank you, this makes me realize we should probably use `smartLink()` here. Can you try locally if this works?

```suggestion
            label: web.smartLink(`/${env.locale}/docs/Web/CSS/CSS_animated_properties`, null, localize(localStrings, "animationType"))
```

Reply:
it worked just fine , thanks for the suggestion